### ISSUE TITLE
Spacy logging messages are duplicated

### ISSUE THREAD

**Issue by nsorros** [wellcometrust/WellcomeML#282]
If you run `python examples/spacy_classifier.py`

you get
```
[2021-04-27 12:01:00,234] [INFO] Added vocab lookups: lexeme_norm
2021-04-27 12:01:00 spacy INFO: Added vocab lookups: lexeme_norm
[2021-04-27 12:01:00,235] [INFO] Created vocabulary
2021-04-27 12:01:00 spacy INFO: Created vocabulary
[2021-04-27 12:01:00,235] [INFO] Finished initializing nlp object
2021-04-27 12:01:00 spacy INFO: Finished initializing nlp object
```

**Comment by aCampello** [comment]
I'm not observing this issue anymore after that PR. Can you confirm?

**Comment by aCampello** [comment]
Yes, this is solved by that PR, and also by #292. Let me know if you observe it again.
